### DISCUSSION TITLE
How to update wallet's logo as the walletconnect-registry is readonly

### DISCUSSION THREAD

**Discussion by chendatony31** [WalletConnect/walletconnect-monorepo#988] [Developer Support] [answered]
Hi, Our TokenPocket wallet's logo has been updated, Where can I submit our new logo?

BTW,  our new logo's URL is https://tp-statics.tokenpocket.pro/TokenPocket_App__white.png  or check it on our site: https://tokenpocket.pro

**Comment by chadyj** [comment, marked as answer]
To update your registry listing simply create an account at WalletConnect.com.
I'll transfer the existing listing to your new account. Just ping me here when you have created an account, and I'll help with next steps.

**Reply by chendatony31** [reply]
My account:  chendatony31@gmail.com

**Reply by chendatony31** [reply]
@chadyj   Hi, I've sent an email using chenda@tokenpocket.pro. Have you received it?

**Reply by chadyj** [reply]
Hey @chendatony31 I didn't get the email, but lets follow up on Discord
